docs/sources/developers/plugins/plugin.schema.json

```
@@ -361,28 +361,32 @@
             "properties": {
               "url": {
                 "type": "string",
-                "description": "For data source plugins. URL to fetch the authentication token."
+                "description": "URL to fetch the authentication token."
+              },
+              "scopes": {
+                "type": "string",
+                "description": ""
               },
               "params": {
                 "type": "object",
-                "description": "For data source plugins. Parameters for the token authentication request.",
+                "description": "Parameters for the token authentication request.",
                 "additionalProperties": false,
                 "properties": {
                   "grant_type": {
                     "type": "string",
-                    "description": "For data source plugins. OAuth grant type."
+                    "description": "OAuth grant type"
```

```suggestion
                    "description": "OAuth grant type."
```
If one description has a period, they all do.